src/components/deposit/LotSizeSelector.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import classNames from 'classnames'
+
+import Check from '../svgs/Check'
+
+const LotSizeOption = ({ lotSize, onClick, selected }) => {
+  const handleClick = () => {
+    onClick(lotSize)
+  }
+  return (
+    <li className={classNames('lot-size-option', { selected })}
+      onClick={handleClick}>
+      <button>
```

Comment:
Not a blocker, but I would represent this as a set of radio buttons. That would also let you leverage the `:checked` pseudoclass to handle rendering the check, and helps screen readers more clearly identify design the intent.